### ISSUE TITLE
[BUG] bottom text is broken on mobile

### ISSUE THREAD

**Issue by B0pol** [darkreader/darkreader#3374]
### **Describe the bug**

On mobile (Firefox), message on the bottom is broken

### **Steps to reproduce**

Open dark reader

### **Expected behavior**

The two lines at the bottom are not mixed together

Therefore we would see 

Version 5 preview (4.9.16)
This project is sponsored by you

Or 
This project is sponsored by you
Version 5 preview (4.9.16)

??
### **Actual behavior**

The two lines are mixed together, see the screenshot below

### **Screenshots**

![Screenshot_20200810-090404_Firefox_Nightly](https://user-images.githubusercontent.com/58657617/89759800-45b6d080-daeb-11ea-9129-51cc9e53ec2f.png)

### **System Information:**
 - OS: Android 9
 - Browser: Fennec F-Droid (68.11), Firefox nightly 200809 06:01
 - Dark Reader version: 4.9.16

### **Additional context**

**Comment by Gusted** [comment]
Yeah it was only tested in Browsers UI. I'm not sure what happened to mobile. It seems like Fenix rendered it a bit weird.
